README.md

```diff
@@ -351,7 +354,207 @@ The last step will launch Jenkins tests. If these pass:
 2. Push the tag: `git push upstream --tags`
 3. Make a PR to [VCM/gt4py](https://github.com/vulcanclimatemodeling/fv3core) that updates the version in `docker/Makefile` to the new tag.
 
-
 ## License
-
 FV3Core is provided under the terms of the [GPLv3](https://www.gnu.org/licenses/gpl-3.0.en.html) license.
+
+# Development guidelines
+
+## File structure / conventions
+The main functionality of the FV3 dynamical core, which has been ported from the
+Fortran version in the fv3gfs-fortran repo, is defined using GT4py stencils and python
+'compute' functions in fv3core/stencils. The core is comprised of units of calculations
+defined for regression testing. These were initially generally separated into distinct files in
+fv3core/stencils with corresponding files in tests/translate/translate_<unit>.py
+defining the translation of variables from Fortran to Python. Exceptions exist in cases
+where topical and logical grouping allowed for code reuse. As refactors optimize the
+model, these units may be merged to occupy the same files and even methods/stencils, but
+the units should still be tested separately, unless determined to be redundant.
```

Comment:
This newline behavior matches what's done in the rest of the document. Having each sentence on its own line makes change logs easier to read, and avoids needing to move text between lines to change line lengths.

```suggestion
The main functionality of the FV3 dynamical core, which has been ported from the Fortran version in the fv3gfs-fortran repo, is defined using GT4py stencils and python 'compute' functions in fv3core/stencils.
The core is comprised of units of calculations defined for regression testing.
These were initially generally separated into distinct files in fv3core/stencils with corresponding files in tests/translate/translate_<unit>.py defining the translation of variables from Fortran to Python.
Exceptions exist in cases where topical and logical grouping allowed for code reuse.
As refactors optimize the model, these units may be merged to occupy the same files and even methods/stencils, but the units should still be tested separately, unless determined to be redundant.
```

Reply:
Please merge these. You can add them to a batch to merge them in one commit. Thanks for fixing the rest of the lines below!